ait-extension/src/jsx/ppro/ppro-utils.ts

```
@@ -0,0 +1,418 @@
+  }
+  let xmp = new XMPMeta(projectItem.getProjectMetadata());
+  for (var i = 0; i < data.length; i++) {
+    let item = data[i];
+    var successfullyAdded = app.project.addPropertyToProjectMetadataSchema(
+      item.fieldName,
+      item.fieldId,
+      2
+    );
+  }
+  var array = [];
+  for (var i = 0; i < data.length; i++) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)